server/bert_serving/server/graph.py

```diff
@@ -76,6 +77,15 @@ def optimize_graph(args, logger=None):
                 token_type_ids=input_type_ids,
                 use_one_hot_embeddings=False,
                 use_position_embeddings=not args.no_position_embeddings)
+            
+            if args.pooling_strategy == PoolingStrategy.CLASSIFICATION:
+                hidden_size = 768
```

Comment:
better use it as variable rather than a magic number

```suggestion
                hidden_size = model.pooled_output.shape[-1].value
```